site/en/docs/extensions/reference/commands/index.md

```diff
@@ -246,7 +240,7 @@ keyboard shortcut.
 ```
 
 ```js
-// background.js
+// service-worker.js
```

Comment:
Can we convert this comment to a [label](https://developer.chrome.com/docs/handbook/components/#labels)?